ARExperiment/ARExperiment.xcodeproj/xcuserdata/berta.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist

```diff
@@ -10,11 +10,11 @@
             ignoreCount = "0"
             continueAfterRunningActions = "No"
             filePath = "ARExperiment/ViewController.swift"
-            timestampString = "536761136.911963"
+            timestampString = "568304874.2774121"
             startingColumnNumber = "9223372036854775807"
             endingColumnNumber = "9223372036854775807"
-            startingLineNumber = "102"
-            endingLineNumber = "102"
+            startingLineNumber = "94"
```

Comment:
just as an FYI, `xcuserdata/` should be in the gitignore file, as these files are specific to the developer.

Reply:
I though it was, let me check 👍